catalyst/runners/runner.py

```diff
@@ -670,6 +670,52 @@ def on_loader_end(self, runner):
         for batch in loader:
             yield self.predict_batch(batch)
 
+    def evaluate_loader(
+        self,
+        model: Model,
```

Comment:
```suggestion
        model: Model = None,
```
we should not respecify the model after `.train`, isn't it?

Reply:
@leoromanovich could you check this out too?

Reply:
Yeah. 
Do you mean, that if a model is None (only callbacks and loader are given), we need simply take the model from the runner state? 
Like:
```python
if model is None:
    model = self.model
```

Reply:
@Scitator In the current version, if a user explicitly changes the model, it will affects on a global state (for example, if after that user runs `predict_loader`, a `runner` will use the model given in `evaluate_loader`). I'm not sure if it's proper behavior. 
What do you think?

Reply:
For now, every time you pass a new model to the Runner parameters, it wraps its behavior, so it looks okay.
Nevertheless, that's why I suggest placing `model=None` by default and check `self.model is not None` to drive a user to one-time model specification for the Runner (we have an init option, btw :) )

Reply:
I've added this to runner and fixed docs wrt it =)